docs/sources/tempo/setup/set-up-test-app.md

```diff
@@ -17,6 +17,7 @@ You'll need:
 
 * Grafana 9.0.0 or higher
 * Microservice deployments require the Tempo querier URL, for example: `http://query-frontend.tempo.svc.cluster.local:3200`
+* [OpenTelemetry telmemetrygen](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/cmd/telemetrygen) for generating tracing data
```

Comment:
We use [xk6-client-tracing](https://github.com/grafana/xk6-client-tracing) in all of our examples. I'm not opposed to the OTel telemetrygen, but wanted to point it out in case we prefer sticking to only that one.

Reply:
I've a couple of comments here:
* For a short 'Is this working' test, `telemtrygen` seems a good choice because it's just testing whether trace ingestion is working correctly, to allow you to quickly search in Grafana for traces. The pipeline works, you can now hook it up to whatever you want. With `xk6-client-tracing` you need to start running a Docker container, and then stop it later. You also need to configure it. 
* The first thing you see if you go to the `xk6-client-tracing` repo is:

    ⚠️ In Development

    This project is in development and changes a lot between commits. Use at your own risk.

I'm more than happy with using it, but do we still feel this isn't stable, can we remove this comment, or does it still apply? The last thing we want to do is provide documentation on validating your Tempo/GET/GCT pipeline and then discovering that the container isn't working for some reason. I'm a little concerned that it implies that we're officially suggesting something that might not be stable.

Thank you!

Reply:
I've opened a doc issue to add the xk6 section to the Set up a test app: https://github.com/grafana/tempo-squad/issues/315

Reply:
Those are fair points. Agree that `telemetrygen` sounds like the best option in here.